### PR TITLE
Add closed schools to search results caution flag filter

### PR DESCRIPTION
**Pull request by hughes-dustin**
## Description
https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/va.gov-team/8187

Filter out school closings along with caution flags when excluding caution flags.

## Acceptance criteria
- [x] When the user selects the filter, institutions with either caution flags OR a school closing flag are excluded from the search results.

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by zurbergram — CHANGES_REQUESTED** [review]
Why not name it `excludeWarnings` instead of `excludeCautionAndSchoolClosureFlags` ?

**Review by shawkey-daniel — APPROVED** [review]
LGTM

**Review by zurbergram — APPROVED** [review]

**Review by sonntag-adam — APPROVED** [review]

File: app/controllers/v0/institution_programs_controller.rb [hunk 1]
```diff
@@ -81,7 +81,14 @@ def filter_results(relation)
         relation = relation.filter(filter_args[0], @query[filter_args[1]])
       end
 
-      relation = relation.where(institutions: { count_of_caution_flags: 0 }) if @query[:exclude_caution_flags]
+      if @query[:exclude_warnings]
+        # prod flag for bah-8187. Remove are approval.
```
Comment (erluti) [inline app/controllers/v0/institution_programs_controller.rb:85]: Could you clarify this comment for me?

Comment (hughes-dustin) [reply app/controllers/v0/institution_programs_controller.rb]: Might have been a typo. The idea was to keep us from using that filter in prod. But I just realized that we still need the old filter so I've added it back in and removed the prod flag .

**Review by philipwritescode — APPROVED** [review]
looks good

**Pull request merged**